RELEASING.md

```diff
@@ -191,6 +191,11 @@ changie new --kind "Dependencies" --body "Bump Engine to $ENGINE_VERSION" --cust
 changie batch patch
 changie merge
 
+cd ../php
+changie new --kind "Dependencies" --body "Bump Engine to $ENGINE_VERSION" --custom "Author=github-actions" --custom "PR=${BUMP_ENGINE_PR:?must be set}"
+changie batch patch
+changie merge
+
```

Comment:
I don't remember us talking about publishing PHP part of our release process. Since we don't do it for any other experimental SDK apart from Elixir thanks to @wingyplus' hard work wiring it all up, my proposal would be to not do it. WDYT @helderco?

Reply:
Woops, I started drafting this, and accidentally left it in, my bad.